.gitignore

```diff
@@ -54,3 +54,7 @@ web-ext-artifacts
 pkg
 release-utils/config.sh
 release-utils/xpi
+
+#Eclipse project stuff
```

Comment:
Could we move these exclusions to somewhere above Privacy Badger-specific exclusions (`release-utils`, etc.)?